sql/gen-sql-markdown.py

```diff
@@ -15,12 +15,18 @@
 # limitations under the License.
 #
 
-import sys
 import os
+import re
+import sys
 from collections import namedtuple
+from textwrap import dedent
+
+from mkdocs.structure.pages import markdown
```

Comment:
@nchammas, do you mind if I ask to add a short comment that it intentionally uses `markdown` in `mkdocs` to reduce the overhead of maintaining dependencies?